schemas/app-sre/saas-file-2.yml

```diff
@@ -127,23 +222,42 @@ properties:
       - url
       - path
       - targets
+
   takeover:
     type: boolean
+    description: |
+      If true, only resources declared in this SaaS file will be kept;
+      all others will be deleted. Use with caution.
+
   deprecated:
     type: boolean
-    description: prevent updates to a saas file
+    description: |
+      If true, prevents updates to this SaaS file.
+
   compare:
     type: boolean
+    description: |
+      If false, the job does not compare desired to current resources and
+      applies all resources even if they have not changed.
+
   timeout:
     "$ref": "/common-1.json#/definitions/dhmsDuration"
     description: |
-      Maximum time the deployment job is allowed to last. In Tekton it defaults to
-      60 minutes. We don't allow smaller times as Tekton implementation currently deletes
-      PipelineRun pods, hence destroying logs.
+      Maximum time the deployment job is allowed to last. Defaults to 60
+      minutes for Tekton. Shorter times are not allowed due to Tekton
+      implementation details.
+
   publishJobLogs:
     type: boolean
+    description: |
+      If true, publishes job logs as artifacts in the deployment job.
```

Comment:
not exactly. It will gather the logs of the `Job` or `ClowdJobInvocation` being deployed and print them out in the pipelinerun output.

https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/openshift_saas_deploy.py#L266-L271